src/frontend/Ast.ml

```diff
@@ -137,6 +140,7 @@ type ('e, 's, 'l, 'f) statement =
   | Print of 'e printable list
   | Reject of 'e printable list
   | Skip
+  | Blank
```

Comment:
I think this would be a helpful place to leave a comment explaining the meaning and usage of Blank